### DISCUSSION TITLE
How to cite aiogram in paper?

### DISCUSSION THREAD

**Discussion by mishagrol** [aiogram/aiogram#1334] [Q&A] [answered]
Hello! 👋

How to cite AioGram in science paper? 

Something like pandas (http://citebay.com/how-to-cite/pandas/) or scipy (https://scipy.org/citing-scipy/)?

**Comment by JrooTJunior** [comment, marked as answer]
Currently, I don't have a recommendation for how to cite an aiogram in a science paper, but you can use following examples.

Note that citations can reference documentation or source code and be formatted in different styles, so, i'll provide many possible variants.

Please change version and release date and view date to version that you are using in your job.

## If you referencing documentation

### APA Style:

> Onufriichuk O. (2023). aiogram: A modern and fully asynchronous framework for Telegram Bot API written in Python  (Version 3.1.1) [Software]. Documentation. https://docs.aiogram.dev/en/latest/

### Vancouver Style:

> Onufriichuk O. 2023. aiogram: A modern and fully asynchronous framework for Telegram Bot API written in Python  (Version 3.1.1) [Software]. Documentation. Available at: https://docs.aiogram.dev/en/latest/

### Harvard Style:

> Onufriichuk O. 2023, 'aiogram: A modern and fully asynchronous framework for Telegram Bot API written in Python ', Documentation, Version 3.1.1, viewed 9 October 2023, https://docs.aiogram.dev/en/latest/.


## Also if you are referencing GitHub sources

### APA Style:

> Onufriichuk O. (2023). aiogram: A modern and fully asynchronous framework for Telegram Bot API written in Python (Version 3.1.1) [Software]. GitHub. https://github.com/aiogram/aiogram

### Vancouver Style:

> Onufriichuk O. 2023. aiogram: A modern and fully asynchronous framework for Telegram Bot API written in Python (Version 3.1.1) [Software]. GitHub. Available at: https://github.com/aiogram/aiogram

### Harvard Style:

> Onufriichuk O. 2023, 'aiogram: A modern and fully asynchronous framework for Telegram Bot API written in Python', GitHub, Version 3.1.1, viewed 9 October 2023, https://github.com/aiogram/aiogram.
